src/main/groovy/life/qbic/datamodel/dtos/business/services/ExternalServiceProduct.groovy

```diff
@@ -0,0 +1,29 @@
+package life.qbic.datamodel.dtos.business.services
+
+import groovy.transform.EqualsAndHashCode
+import life.qbic.datamodel.dtos.business.ProductId
+import life.qbic.datamodel.dtos.business.facilities.Facility
+
+/**
+ * <p>External service product</p>
+ *
+ * <p>Represents a service product that is offered through QBiC by an external partner.</p>
+ *
+ * @since 2.13.0
+ */
+@EqualsAndHashCode(callSuper = true)
+class ExternalServiceProduct extends PartialProduct {
```

Comment:
Why is it a partial product? I would see it similar as Sequencing. I thought only PM and DS can be partial?

Reply:
I want to leave it open, since we dont know what external services will be